api/src/main/java/net/kyori/adventure/audience/ForwardingAudience.java

```diff
@@ -0,0 +1,65 @@
+package net.kyori.adventure.audience;
+
+import net.kyori.adventure.bossbar.BossBar;
+import net.kyori.adventure.sound.Sound;
+import net.kyori.adventure.sound.SoundStop;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.title.Title;
+import org.checkerframework.checker.nullness.qual.NonNull;
+
+/**
+ * An audience that forwards all methods to another audience.
+ */
+public interface ForwardingAudience extends Audience {
+  /**
+   * Gets the forwarded audience.
+   *
+   * @return the forwarded audience
+   */
+  @NonNull Audience audience();
```

Comment:
It might be useful for this to be `@Nullable`, then silently fail if `null`. Or maybe define an `Audience.empty()` to do the same thing?

Reply:
https://github.com/KyoriPowered/text/pull/62/commits/fd7ce1a567b36ac5cda35207c90354e398777f5d